plugins/searchdocs.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+from errbot import BotPlugin, botcmd
+
+
+class Searchdocs(BotPlugin):
+    """
+    Search API and user docs
+    """
+
+    API_DOCS = 'http://api.coala.io/en/latest'
+    USER_DOCS = 'http://docs.coala.io/en/latest'
+
+    @botcmd
+    def search(self, msg, arg):
```

Comment:
The doc comment should certainly contain the correct inputs expected by this method. As in intended behavior in order to be able to evaluate the regexes.